arbstate/inbox.go

```diff
@@ -41,6 +44,26 @@ type MessageWithMetadata struct {
 	DelayedMessagesRead uint64                   `json:"delayedMessagesRead"`
 }
 
+var EmptyTestMessageWithMetadata = MessageWithMetadata{
+	Message: &arbos.L1IncomingMessage{
+		Header: &arbos.L1IncomingMessageHeader{},
+	},
+}
```

Comment:
I could add extra checks in the code to ensure it doesn't try and use a nil `L1IncomingMessage` or nil `L1IncomingMessageHeader`.  I figured it would be easier to just have a populated test item since it wouldn't happen in prod